apps/omg_api/test/block_queue/core_test.exs

```diff
@@ -486,43 +492,43 @@ defmodule OMG.API.BlockQueue.CoreTest do
     test "Gas price is lowered and then raised when ethereum blocks gap gets filled", %{
       empty_with_gas_params: empty_with_gas_params
     } do
-      state = empty_with_gas_params
+      state = empty_with_gas_params |> Core.enqueue_block(<<6>>, 6 * @child_block_interval, 1)
       gas_params = %{state.gas_price_adj_params | eth_gap_without_child_blocks: 3}
       state1 = %{state | gas_price_adj_params: gas_params}
 
       {:do_form_block, state2} =
         state1
-        |> set_ethereum_status(2, 0, false)
+        |> set_ethereum_status(4, 5 * @child_block_interval, false)
 
       assert state.gas_price_to_use > state2.gas_price_to_use
+      state2 = state2 |> Core.enqueue_block(<<6>>, 7 * @child_block_interval, 1)
 
-      {:dont_form_block, state3} =
+      {:do_form_block, state3} =
         state2
-        |> set_ethereum_status(3, 0, false)
+        |> set_ethereum_status(6, 5 * @child_block_interval, false)
 
       assert state2.gas_price_to_use > state3.gas_price_to_use
 
       # Now the ethereum block gap without child blocks is reached
-      {:dont_form_block, state4} =
+      {:do_form_block, state4} =
         state2
-        |> set_ethereum_status(4, 0, false)
+        |> set_ethereum_status(7, 5 * @child_block_interval, false)
 
       assert state3.gas_price_to_use < state4.gas_price_to_use
     end
 
     @tag fixtures: [:empty_with_gas_params]
-    test "Gas price calculation cannot be raised above limit", %{empty_with_gas_params: empty_with_gas_params} do
-      state = empty_with_gas_params
+    test "Gas price calculation cannot be raised above limit", %{empty_with_gas_params: state} do
       expected_max_price = 5 * state.gas_price_to_use
       gas_params = %{state.gas_price_adj_params | gas_price_raising_factor: 10, max_gas_price: expected_max_price}
-      state1 = %{state | gas_price_adj_params: gas_params}
-      eth_gap = state1.gas_price_adj_params.eth_gap_without_child_blocks
-
-      {:do_form_block, newstate} =
-        state1
-        |> set_ethereum_status(1 + eth_gap, 0, false)
+      state = %{state | gas_price_adj_params: gas_params} |> enqueue_block(<<0>>, 6 * @child_block_interval, 1)
 
-      assert expected_max_price == newstate.gas_price_to_use
+      # many times not main block
```

Comment:
Despite Ethereum height changing multiple times, gas price does not grow since no new blocks are mined

Reply:
Also, isn't this behavior tested below?